ngraph_bridge/ngraph_utils.cc

```diff
@@ -469,7 +463,7 @@ bool IsProcessedByNgraphPass(Graph* g) {
   // Current method may fail when graph has no encapsulates after first pass
   // Also variable/optimizer change introduces other types of ng nodes
   for (Node* node : g->nodes()) {
-    if (node->type_string() == "NGraphEncapsulate") return true;
+    if (node->type_string() == "_nGraphEncapsulate") return true;
```

Comment:
Comment at line 464 can be removed, we don't have variables/optimizers anymore

Reply:
Removed.